paper-server/src/main/java/org/bukkit/craftbukkit/block/CraftBiome.java

```diff
@@ -1,5 +1,12 @@
 package org.bukkit.craftbukkit.block;
 
+import java.util.Objects;
```

Comment:
This import has been duplicated and those could be sorted too (applies to all the classes you changed)